docs/2018-11-08-Configuration-Splitting-Simplification.md

```diff
@@ -6,11 +6,9 @@ Background
 
 ### Functional and function-like patterns
 
-A functional pattern has exactly one element, i.e. `t` is functional iff
-`∃x . x=t`. A function-like pattern is similar, but it can also be `⊥`, i.e.
-`t` is function-like if `(∃x . x=t) ∨ ⌈¬t⌉`.
+See the `glossary.md` file for term definitions.
```

Comment:
I wonder whether here you could have an anchor like `glossary.md#functionlike` :-)

Reply:
We can, done.